src/ralph/deployment/models.py

```diff
@@ -80,6 +159,28 @@ class Deployment(TransitionJob):
     class Meta:
         proxy = True
 
+    @classmethod
+    def get_deployment_for_ip(cls, ip):
+        base_object = Ethernet.objects.get(ipaddress__address=ip).base_object
+        deployment = None
+        try:
+            deployment = cls.objects.active().get(
+                content_type_id=base_object.content_type_id,
+                object_id=base_object.id
+            )
+        except cls.DoesNotExist:
```

Comment:
add TODO that there shoulb be validation for existing deployment for the same IP